### ISSUE TITLE
Order homepage like a table of contents

### ISSUE THREAD

**Issue by marcusrelacion** [AIGAdc/dcdesignweek.org-11ty#8]
Headings should be sequential, so the different heading levels reflect the hierarchy of different sections of content.

**Comment by marcusrelacion** [comment]
This was fixed while de-branding 2021 DCDW website.
